clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/ClientConfig.java

```diff
@@ -60,6 +60,13 @@
   private boolean isHttps = false;
   private SSLFactory sslFactory = null;
 
+  // HttpTransport settings
+  private int maxConnectionsPerRoute = 2; // AsyncHttpClient Default, only for HTTP1
+
+  private int maxConnectionsTotal = 20; // AsyncHttpClient Default, only for HTTP1
+
+  private boolean requireHTTP2;
```

Comment:
Can this config to be more specific? such as `httpClient5Http2Enabled`? since D2 based impl doesn't read this config.